index.md

```diff
@@ -5,49 +5,20 @@ title: OWASP Code Review Guide
 tags: Code Review Guide
 level: 3
 type: documentation
-auto-migrated: 1
 
 ---
 
-The current PDF version can be found [here](/www-pdf-archive/OWASP_Code_Review_Guide_v2.pdf)
+[![OWASP Flagship](https://img.shields.io/badge/owasp-lab-yellow.svg)](https://owasp.org/projects/#all-projects)
+[![CC BY-SA 3.0](https://licensebuttons.net/l/by-sa/3.0/80x15.png)](https://creativecommons.org/licenses/by-sa/3.0/)

```

Comment:
Let's bump this version 4 and update the license, what say you?

Reply:
That's fine with me